mypy.ini

```diff
@@ -0,0 +1,232 @@
+disallow_untyped_defs = False
+
+[mypy-tests.test_mixins.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_modifiers.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_notebooks.*]
+disallow_untyped_defs = False
+
+[mypy-tests.test_optim.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_paramsets.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_paramviewer.*]
+check_untyped_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_patchset.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_pdf.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_probability.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_public_api.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_public_api_repr.*]
+disallow_incomplete_defs = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_regression.*]
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_schema.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_scripts.*]
+check_untyped_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_simplemodels.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_tensor.*]
+check_untyped_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_tensorviewer.*]
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_teststats.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_toys.*]
+check_untyped_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_utils.*]
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_validation.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+[mypy-tests.test_workspace.*]
+check_untyped_defs = False
+disallow_incomplete_defs = False
+disallow_untyped_calls = False
+disallow_untyped_defs = False
+
+
+# Third-party module rule relaxations
+[mypy-iminuit.*]
+ignore_missing_imports = True
```

Comment:
As I discovered via @henryiii, these ignores can be specified in a nice and compact manner in `pyproject.toml`, see https://github.com/scikit-hep/hist/pull/230.